### PR TITLE
Pass error style as a prop to TextField

### PR DESCRIPTION
**Pull request by shawngustaw**

### CODE REVIEW

File: src/PasswordField.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ class PasswordField extends React.Component {
         <div style={styles.inputContainer}>
           <TextField
             {...other}
-            errorStyle={styles.error}
+            errorStyle={errorStyle || styles.error}
```
Comment (leMaik) [inline src/PasswordField.js]: Why not `{{ ...styles.error, ...errorStyle }}` as with `style`? That way, styles would be merged.

**Comment by shawngustaw** [comment]
@leMaik good call!

File: src/PasswordField.js [hunk 2]
```diff
@@ -126,7 +126,7 @@ class PasswordField extends React.Component {
         <div style={styles.inputContainer}>
           <TextField
             {...other}
-            errorStyle={styles.error}
+            errorStyle={...styles.error, ...errorStyle}
```
Comment (leMaik) [inline src/PasswordField.js]: Double brackets are required here. :wink:

**Comment by leMaik** [comment]
@shawngustaw Thanks for the PR, I'm happy that atleast _somebody_ uses this component. :)

**Comment by shawngustaw** [comment]
Pretty useful component so thanks for open sourcing it!

**Comment by leMaik** [comment]
Released in v1.0.4 (and v1.0.3).

**Pull request merged**